Apps/Sandcastle/gallery/development/3D Models Outline.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
```

Comment:
@lilleyse it could be cool to combine this example with #4547 and then fade in/out both coloring the model and exlarging the silhouette over time.  Tweens are not part of the [public API](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/TweenCollection.js) so we might want to expose them if time permits.